Release 5.2.1 --> Update CHANGELOG and package version. Prepare for release. (#199)
blackbaud/skyux-ag-grid: CHANGELOG.md

# 5.2.1 (2022-02-07)

> - Updated code structure for Data grid with data manager & Data entry grid with data manager's code examples and behavioral demos. Made sorts not be segemented by selection. Remove unnecessary variable 'searchText' from code examples. Make shown items update dynamically based on selection. [#194](https://github.com/blackbaud/skyux-ag-grid/pull/194)

Reviewer: ```suggestion
- Updated the code structure for data grid and data entry grid code examples and behavioral demos to not segment by selection, remove the unnecessary `searchText` variable, and update the shown items dynamically based on selection. [#194](https://github.com/blackbaud/skyux-ag-grid/pull/194)
```

Author: fixed in commit [`800bf9e` - update changelog](https://github.com/blackbaud/skyux-ag-grid/pull/199/commits/800bf9ead565ce7f1bd0bdd24dd4def26cef01a4)
